handy/logging.cc

```diff
@@ -76,8 +76,10 @@ void Logger::setFileName(const string& filename) {
 }
 
 void Logger::maybeRotate() {
+  //以第一次设置的日志文件名为日志基准文件名
+  static std::string basename = filename_;
     time_t now = time(NULL);
-    if (filename_.empty() || (now - timezone) / rotateInterval_ == (lastRotate_ - timezone)/ rotateInterval_) {
+    if (filename_.empty() || (now - timezone) / rotateInterval_ == (lastRotate_ - timezone) / rotateInterval_) {
```

Comment:
The condition in `if` exp is too confused to read it clearly.
Could you write it in an independent line?

Reply:
the interval of updating log filename is rotateInterval_, the condition in if exp is means that the interval between in two updates whether greater the rotateInterval_.

Reply:
I meant that for more readability, we should write it like this:
```c++
bool xxxxx =  (now - timezone) / rotateInterval_ == (lastRotate_ - timezone) / rotateInterval_;
if (filename_.empty() ||xxxxx) {
//...
```